conda/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+    name: odlcuda
+    version: "0.3.0"
+
+source:
+    git_url: https://github.com/odlgroup/odlcuda.git
+    git_rev: conda-build
+
+build:
+    script_env:
+        - CUDA_COMPUTE
+
+requirements:
+    build:
+        - python 2.7|>=3.4
+        - nomkl
+        - setuptools
+        - numpy >=1.9
+        - future >=0.14
+        - cudatoolkit
+        - boost
+        # Make sure to run "conda config --append channels odlgroup"
+        - odl >=0.3.0
```

Comment:
I don't think we want to keep the version perfectly in sync, so probably a version after the split is sufficient. It will anyway pick the most recent one.